test/transaction_builder.js

```diff
@@ -477,7 +477,7 @@ describe('TransactionBuilder', function () {
                   redeem: payments.p2ms({
                     output: redeemScript,
                     signatures
-                  }, { allowIncomplete: true })
+                  }, { allowIncomplete: true, validate: false })
```

Comment:
wait, shouldn't `allowIncomplete` allow for `validate: true`,  but in an acceptable way?
They aren't exclusive?